sycl/test/basic_tests/vec_convert.cpp

```diff
@@ -149,6 +151,5 @@ int main() {
   test<float, half, 8, rounding_mode::rtn>(
       float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f},
       half8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f});
-
   return 0;
-}
+}
```

Comment:
Don't see much value in these changes.
Suggest keeping original code.

Reply:
Only part of this comment was resolved.

Reply:
I don't understand, why they are different?

Reply:
You seem to delete line ending character.